### DISCUSSION TITLE
Argocd vault plugin not working

### DISCUSSION THREAD

**Discussion by osahenry11** [argoproj/argo-cd#18275] [General]
I deployed a the argocd vault plugin using the side car and init container with config map of the plugins implementation. I can see it through my Argo cd UI but when I go to create application and sync it gives me an error. 

Unable to create application: application spec for example-secret is invalid: InvalidSpecError: Unable to generate manifests in ./argocd-vault-plugin-demo-main/dog/: rpc error: code = Unknown desc = plugin sidecar failed. error generating manifests in cmp: rpc error: code = Unknown desc = error generating manifests: fork/exec /usr/local/bin/argocd-vault-plugin: exec format error

**Comment by kbeham17** [comment]
I currently suffer with exactly the same problem. 

`level=error msg="finished streaming call with code Unknown" error="error generating manifests: fork/exec /usr/local/bin/argocd-vault-plugin: exec format error" grpc.code=Unknown grpc.method=GenerateManifest`

Were you able to fix it?
